linera-views/src/dynamo_db.rs

```diff
@@ -1588,7 +1602,7 @@ pub async fn create_dynamo_db_test_client() -> DynamoDbClient {
 }
 
 /// Helper function to list the names of tables registered on DynamoDB.
-pub async fn list_tables(
+pub async fn get_list_tables(
```

Comment:
Why change to `get_list_tables`? (which is not correct English)

Reply:
It is because first of all the `list_tables` is the name I want to keep.

But if I kept it, then it would collide with the existing "list_tables". So, there is a need for rename. The `get_list_tables` is incorrect as you point out. And so I put the name `list_tables_from_client`.

Reply:
I see. I left some suggestions but I guess `list_tables_from_client` works for now